### PR TITLE
[ACA-4384] [ADW] Empty template or error template should be displayed in case error occurred when clicking a folder

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACA-4384

We were not showing an error template in case of BE 500 error 

**What is the new behaviour?**

image

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/app/components/files/files.component.spec.ts [hunk 1]
```diff
@@ -106,7 +107,7 @@ describe('FilesComponent', () => {
     spyContent.and.returnValue(of({ entry: node }));
   });
 
-  describe('Current page is valid', () => {
+  fdescribe('Current page is valid', () => {
```
Comment (mauriziovitale) [inline src/app/components/files/files.component.spec.ts:110]: be careful

File: src/app/components/files/files.component.spec.ts [hunk 2]
```diff
@@ -128,6 +129,21 @@ describe('FilesComponent', () => {
 
       expect(component.isValidPath).toBe(true);
     }));
+
+    it('should set current page as invalid path when loadFolderByNodeId API fails with 500 status code', fakeAsync(() => {
+      debugger;
```
Comment (mauriziovitale) [inline src/app/components/files/files.component.spec.ts:134]: debugger?

File: src/app/components/files/files.component.spec.ts (hunk 1 above)
Comment (sivakumar414ram) [reply src/app/components/files/files.component.spec.ts]: Updated

**Review by eromano — APPROVED** [review]

File: src/app/components/files/files.component.ts [hunk 3]
```diff
@@ -344,4 +344,14 @@ export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
   isFilterHeaderActive(): boolean {
     return this.showHeader === ShowHeaderMode.Always;
   }
+
+  onError(error: any) {
+    if (error?.message) {
```
Comment (mauriziovitale) [inline src/app/components/files/files.component.ts:349]: don't think we need to check only 500 error. let's have the template for every error and in the future we can create specific template based on the status if needed

Comment (sivakumar414ram) [reply src/app/components/files/files.component.ts]: Updated

**Review by mauriziovitale — APPROVED** [review]

**Pull request merged**